runtimes/alibi-detect/mlserver_alibi_detect/runtime.py

```diff
@@ -74,9 +74,30 @@ async def predict(self, payload: types.InferenceRequest) -> types.InferenceRespo
             outputs.append(
                 NumpyCodec.encode_output(name=key, payload=np.array([y["data"][key]]))
             )
+        # Add headers
+        y["meta"]["headers"] = {
+            "x-seldon-alibi-type":self.getAlibiType(),
+            "x-seldon-alibi-method": self.getAlibiMethod()
+        }
         return types.InferenceResponse(
             model_name=self.name,
             model_version=self.version,
             parameters=y["meta"],
             outputs=outputs,
         )
+
+    def getAlibiMethod(self) -> str:
```

Comment:
nit: could we use `snake_case` instead of `camelCase` to align with Python conventions?